api/db/changelog/db.changelog-199-update-demographic-survey-v2.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="thibault" id="changelog-199-update-demographic-survey-v2">
+    <addColumn tableName="demographic_survey_v2">
+      <column name="ethnicity_black_other_text" type="VARCHAR(200)"/>
+      <column name="ethnicity_hispanic_other_text" type="VARCHAR(200)"/>
+      <column name="ethnicity_me_na_other_text" type="VARCHAR(200)"/>
+      <column name="ethnicity_nh_pi_other_text" type="VARCHAR(200)"/>
+      <column name="ethnicity_white_other_text" type="VARCHAR(200)"/>
+    </addColumn>
+    <sql>
+      ALTER TABLE demographic_survey_v2_ethnic_category
+        MODIFY COLUMN ethnic_category
+        ENUM(
+          'AI_AN',
```

Comment:
Should match the lists at:
* DbDemographicSurvey.DbEthnicCategory
* workbench-api.yaml
* rdr.yaml